test/deprecated/c_str.good

```diff
@@ -0,0 +1,2 @@
+my string
+my bytes
```

Comment:
I think you need to move this to test/unstable?

Reply:
yes, moved